src/raftstore/store/snap.rs

```diff
@@ -879,6 +884,7 @@ impl Snapshot for Snap {
             // Check each cf file has been fully written, and the checksum matches.
             {
                 let mut file = cf_file.file.take().unwrap();
+                limiter.request(cf_file.size as i64);
```

Comment:
Most content of this file has been written to disk here, so the `limiter` here can't play the role of limiter.

Reply:
Fixed.